Casks/l/locu.rb

```diff
@@ -0,0 +1,32 @@
+cask "locu" do
+  arch arm: "arm64"
+
+  version "0.22.8"
+  sha256 arm: "f82b48ec337fd4f8ee373d4feba23b8d740a7cde72fa8c659d12c1323ee4926c"
+
+  url "https://locu.sfo2.digitaloceanspaces.com/Locu-#{version}-#{arch}-mac.zip",
```

Comment:
If we incorporate the `-` into the `arch` we can adjust as below and the Intel version should now work.

```suggestion
  url "https://locu.sfo2.digitaloceanspaces.com/Locu-#{version}#{arch}-mac.zip",
```